arch/generic/compare256_c.c

```
@@ -4,19 +4,74 @@
-#  define COMPARE256          compare256_8
-#elif defined(HAVE_BUILTIN_CTZLL)
-#  define COMPARE256          compare256_64
-#elif defined(HAVE_BUILTIN_CTZ)
-#  define COMPARE256          compare256_32
+#  define COMPARE256 compare256_8_static
 #else
-#  define COMPARE256          compare256_16
+#  define COMPARE256 compare256_64_static
 #endif
 
+Z_INTERNAL uint32_t compare256_8(const uint8_t *src0, const uint8_t *src1) {
```

Cppcheck (2.19.0)

[style] The function 'compare256_8' is never used.

(unusedFunction)